release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -405,6 +405,16 @@ To manage the cloud controller manager and cloud node manager deployments and li
 
 For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Platform Operators reference_.
 
+[discrete]
+[id="ocp-4-13-mco-certificate-changes"]
+==== The MCO now syncs kubelet CA certificates on paused pools
+
+Previously, the Machine Config Operator (MCO) updated the kubelet client certificate authority (CA) certificate, `/etc/kubernetes/kubelet-ca.crt`, as part of the regular machine config update. Starting with {product-title} (product-version}, the `kubelet-ca.crt` no longer gets updated as part of the regular machine config update. As a result of this change, the Machine Config Daemon (MCD) automatically keeps the `kubelet-ca.crt` up to date whenever changes to the certificate occur. 
+
+Also, if a machine config pool is paused, the MCD is now able to push the newly rotated certificates to those nodes. A new rendered machine config, which contains the changes to the certificate, is generated for the pool, like in previous versions, and the pool still indicates that an update is required. However, as the certificate is updated separately, it is safe to keep the pool paused, assuming there are no further updates. This functionality will be removed in a future release of this product.
```

Comment:
```suggestion
Also, if a machine config pool is paused, the MCD is now able to push the newly rotated certificates to those nodes. A new rendered machine config, which contains the changes to the certificate, is generated for the pool, like in previous versions, and the pool still indicates that an update is required. However, as the certificate is updated separately, it is safe to keep the pool paused, assuming there are no further updates. This function will be removed in a future release of this product.
```
Consider using 'function because this is not a set of functions. See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f